api/v1/instancemanager.go

```diff
@@ -52,3 +52,7 @@ type ListHostsResponse struct {
 	// paging through out all the results.
 	NextPageToken string `json:"nextPageToken,omitempty"`
 }
+
+type Info struct {
+	RuntimeType string `json:"type"`
```

Comment:
If using `InstanceManagerConfig`, this filed could be  just `Type` instead.

Reply:
Updated: use `Config` as API response & `instanceManagerType` as field name